spec/features/users/sign_in_spec.rb

```diff
@@ -131,14 +131,24 @@
         expect(page).to_not have_content(t('forms.buttons.submit.continue'))
 
         fill_in_credentials_and_click_sign_in(user.email, user.password)
-
         expect(page).to have_content t('errors.invalid_authenticity_token')
 
         fill_in_credentials_and_click_sign_in(user.email, user.password)
-
         expect(current_path).to eq user_two_factor_authentication_path
       end
     end
+
+    xit 'displays the session timeout modal, does not allow the user to submit', js: true do
```

Comment:
The modal overlay seems to make accessing or focusing on any of the form elements impossible.  Only thing I could think of is just to double check that the submit button really is disabled: `expect(page).to have_css('[type=submit][disabled]') `